release/ray_release/job_manager/anyscale_job_manager.py

```diff
@@ -56,16 +56,15 @@ def _run_job(
         env = os.environ.copy()
         env.setdefault("ANYSCALE_HOST", str(ANYSCALE_HOST))
 
-        full_cmd = " ".join(f"{k}={v}" for k, v in env_vars.items()) + " " + cmd_to_run
         logger.info(f"Executing {cmd_to_run} with {env_vars} via Anyscale job submit")
 
         anyscale_client = self.sdk
 
-        runtime_env = None
+        runtime_env = {"env_vars": env_vars}
         if working_dir:
-            runtime_env = {"working_dir": working_dir}
-            if upload_path:
-                runtime_env["upload_path"] = upload_path
+            runtime_env["working_dir"] = working_dir
+        if upload_path:
+            runtime_env["upload_path"] = upload_path
```

Comment:
hmm... not sure if this indentation change is desired/correct. could you explain a bit more in details?

Reply:
oh good catch, I should keep the indentation (don't really understand what it does so would just keep them the same)